tests/crypto_crls_cert_chains/data/intermediate2.cnf

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# OpenSSL configuration for test CRL generation
+#
+####################################################################
+[ ca ]
+default_ca = CA_default        # The default ca section
+
+####################################################################
+[ CA_default ]
+database    = ./intermediate_index2.txt
+crlnumber   = ./intermediate_crl_number2
+
+# The root key and root certificate.
+private_key       = ../data/IntermediateCA1.key.pem
```

Comment:
A bit confusing:
intermediate.cnf and intermediate2.cnf uses the same intermediateca1.key and cert.